frontend/dist/views/web/permission-denied.html

```diff
@@ -0,0 +1,36 @@
+<style>
+.button {
+    background-color: #4D4DFF; /* Green */
+    border: none;
+    color: white;
+    padding: 15px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 4px 2px;
+    cursor: pointer;
+    border-radius:8px;
+}
```

Comment:
Hi @16116073, please use existing classes to style the buttons. You can find it in base.scss.

Reply:
done